### ISSUE TITLE
Add a horizontal scrollbar when corp has lots of servers

### ISSUE THREAD

**Issue by SomeSketchyGuy** [mtgred/netrunner#512]
In a game with two players on the corp side the cards will go off the bottom of the screen along with the menu for selecting the servers, making the game unplayable

**Comment by SomeSketchyGuy** [comment]
I just updated chrome so it may have something to do with that since it was working fine before

**Comment by queueseven** [comment]
Have you tried just zooming out?

Could you tell what version of chrome you're running and perhaps provide a screenshot?

**Comment by SomeSketchyGuy** [comment]
Zooming out worked, I believe I'm running the latest version of chrome which is 44.0.2403.130

**Comment by queueseven** [comment]
I have a small laptop and I can only play when setting the zoom level to 90%; that's a good workaround

**Comment by mtgred** [comment]
I also believe it is a Chrome issue. Version 43 broke the game log. Version 44 pushes the bottom of the board outside the window when there are too many content. It also causes #427.

**Comment by zaroth** [comment]
I can confirm that this issue exists in latest Firefox too, I was unable to play as/vs horizontal corp decks since zooming out in Firefox works differently than in Chrome - just the text (Server 0/1/2 etc.) got smaller, not the card graphics themselves. Only after installing Firefox plugin [Zoom Page](https://addons.mozilla.org/en-US/firefox/addon/zoom-page/) I was able to zoom out and see whole board properly.

I tried to dabble in the UI with Firebug to try and fix it, tinkering with *overflow* CSS property, but unfortunately my CSS skills weren't strong enough to make Firefox show the layout properly.
